### DISCUSSION TITLE
buddhism/latest/heart_sutra/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#121] [General]
# buddhism/latest/heart_sutra/

https://shanenull.com/buddhism/latest/heart_sutra/

**Comment by shane0** [comment]
# breakdown

## overview

- prajna paramita
- 5 skhandas
- 6 dusts
- 4 noble truths


## fearless

#### chung tai

- https://ctworld.org.tw/Buddhist%20e-Books/Books05/index.html

```text
with no hindrances there is no fear

- hindrances references below


- citation note

fear comes from misunderstanding and ignorance
with prajna wisdom all fear is removed
```

- https://shanenull.com/buddhism/2023/sanskrit/#five-hindrances

```text
Sensory desire (kamacchanda): The strong attachment to pleasant sensory experiences and desire for more.
Ill-will (vyapada): Feelings of hostility, anger, and aversion towards oneself or others.
Sloth and torpor (thina-middha): A state of dullness or sleepiness that hinders alertness and clarity of mind.
Restlessness and worry (uddhacca-kukkucca): An agitated and restless mind that is distracted by worries, fears, and anxieties.
Doubt (vicikiccha): The lack of faith in oneself or the practice, leading to uncertainty and indecisiveness.
```

- ox herder https://shanenull.com/buddhism/2023/oxherding/

```text
Tamed. The mental hindrances begin to disappear.
```


#### bdk 

- https://bdkamerica.org/product/a-comprehensive-commentary-on-the-heart-sutra/

```text
NO SUFFERING, NO CAUSE, NO CESSATION, AND NO PATH. Ill
THERE IS NO KNOWLEDGE AND NO ATTAINMENT. 113
THEREFORE, BECAUSE OF NON-ATTAINMENT, 115
THE BODHISATTVA, HAVING RELIED ON THE PERFECTION OF
WISDOM, ABIDES WITHOUT MENTAL OBSTRUCTIONS. BECAUSE
OF THE NONEXISTENCE OF MENTAL OBSTRUCTIONS, HE HAS
NO FEAR, HAS OVERCOME ALL PERVERTED VIEWS, HAS LEFT
DREAMLIKE THOUGHTS FAR BEHIND, AND IN THE END
ACHIEVES NIRVANA.
```

## other fear topics

- vajrayana: find the quotes from urgyen about hope & fear
- find the one that was similar to this Fear is the belief that life is safer in the past. Hope is the belief that life is better in the future
- something similar to mental fabrications aversion and clinging fear is rejecting the past hope is rejecting the present or similar

**Reply by shane0** [reply]
# more bdk notes on fear

Fear is of five kinds: 

(1) fear of mortality, arising from the craving
for one’s existence; 

(2) fear of infamy, arising from not benefiting
others and from having desires; 

(3) fear of death, arising from
having the thought of the destruction of one’s self; 

(4) fear of evil transmigrations, arising from not meeting with the Buddhas and
from evil karma; and 

(5) fear of others, arising from seeing oneself
as inferior and others as superior.
